anthos-bm-gcp-terraform/resources/manifests/pos-ingress.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+#################################################################
+# You have to have the Point-of-Sale application deployed first
+# kubectl apply -f \
+#   https://raw.githubusercontent.com/GoogleCloudPlatform/anthos-samples/main/anthos-bm-gcp-terraform/resources/manifests/point-of-sale.yaml
```

Comment:
Not a blocker, but an important note:
You should peg this instruction to a specific tag/release for that field. Otherwise, any changes to the file i the HEAD of the project may be liable to not be compatible with the spec in this file.

Since the references file is also being added in this commit, perhaps the TODO is to include a tag/region after this PR is merged.

Reply:
Thank you for the review. I will update this once I have the manuallb work merged!